layers/relay/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# syntax = docker/dockerfile:1.4
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION:2 AS ext
+
+ARG RELAY_VERSION=0.5.1
+
+# Docs: https://relay.so/docs/1.x/installation#manual-installation
+
+# Install extensions needed by Relay
+RUN pecl install igbinary msgpack
+RUN cp `php-config --extension-dir`/igbinary.so /tmp/igbinary.so
+RUN cp `php-config --extension-dir`/msgpack.so /tmp/msgpack.so
+RUN echo 'extension=igbinary.so' > /tmp/ext-igbinary.ini
+RUN echo 'extension=msgpack.so' > /tmp/ext-msgpack.ini
+
+# Install system libraries needed by Relay
+RUN yum install -y openssl11-libs # libssl 1.1 and libcrypto 1.1 (system version is 1.0)
+
+RUN <<'END' bash -e
+    mkdir -p /tmp/relay
+    export version=$(php -r "echo PHP_MAJOR_VERSION.'.'.PHP_MINOR_VERSION;")
+    curl -sSL "https://cachewerk.s3.amazonaws.com/relay/v$RELAY_VERSION/relay-v$RELAY_VERSION-php$version-centos7-x86-64.tar.gz" | tar -xz --strip-components=1 -C /tmp/relay
```

Comment:
```suggestion
   curl -sSL "https://builds.r2.relay.so/v$RELAY_VERSION/relay-v$RELAY_VERSION-php$version-centos7-x86-64.tar.gz" | tar -xz --strip-components=1 -C
```

Reply:
We just switched over to R2, all future releases will live there. S3 is too expensive at scale.